src/tests/topp/FileFilter_52_output.consensusXML

```diff
@@ -0,0 +1,13344 @@
+		<UserParam type="string" name="parameter: algorithm:FeatureDeconvolution:q_try" value="all"/>
+		<UserParam type="float" name="parameter: algorithm:FeatureDeconvolution:retention_max_diff" value="8.0"/>
+		<UserParam type="float" name="parameter: algorithm:FeatureDeconvolution:retention_max_diff_local" value="25.0"/>
+		<UserParam type="float" name="parameter: algorithm:FeatureDeconvolution:mass_max_diff" value="0.2"/>
+		<UserParam type="stringList" name="parameter: algorithm:FeatureDeconvolution:potential_adducts" value="[H+:0.9]"/>
+		<UserParam type="int" name="parameter: algorithm:FeatureDeconvolution:max_neutrals" value="0"/>
+		<UserParam type="int" name="parameter: algorithm:FeatureDeconvolution:max_minority_bound" value="2"/>
+		<UserParam type="float" name="parameter: algorithm:FeatureDeconvolution:min_rt_overlap" value="0.0"/>
+		<UserParam type="string" name="parameter: algorithm:FeatureDeconvolution:intensity_filter" value="false"/>
+		<UserParam type="string" name="parameter: algorithm:FeatureDeconvolution:default_map_label" value="decharged features"/>
+		<UserParam type="int" name="parameter: algorithm:FeatureDeconvolution:verbose_level" value="0"/>
+		<UserParam type="float" name="parameter: algorithm:FeatureDeconvolution:mass_max" value="25000.0"/>
+	</dataProcessing>
+	<dataProcessing completion_time="1999-12-31T23:59:59">
+		<software name="FileFilter" version="version_string" />
+		<processingAction name="Data filtering" />
+		<UserParam type="string" name="parameter: mode" value="test_mode"/>
+	</dataProcessing>
+	<mapList count="1">
+		<map id="0" name="EU_Gliadinextrakt_100ng_25bis40in90min2.FeatureXML" label="decharged features" size="2092">
+		</map>
+	</mapList>
+	<consensusElementList>
+		<consensusElement id="e_4636468136675691969" quality="0.0">
+			<centroid rt="224.477357142856988" mz="48871.33184998609795" it="1.99883e05"/>
+			<groupedElementList>
+				<element map="0" id="6722364803272170291" rt="224.964699999999994" mz="1577.545300249219963" it="35056.601563" charge="31"/>
+				<element map="0" id="10887204495569992262" rt="224.119200000000006" mz="1397.248683427450032" it="27952.300781" charge="35"/>
+				<element map="0" id="11369039682247725242" rt="224.964699999999994" mz="1746.748917190310067" it="24485.300781" charge="28"/>
+				<element map="0" id="11551657342509961930" rt="224.964699999999994" mz="1630.010463770880051" it="36577.5" charge="30"/>
+				<element map="0" id="13949842513044340834" rt="221.561399999999992" mz="1358.42698779363991" it="21285.900391" charge="36"/>
+				<element map="0" id="14853527260641856532" rt="224.119200000000006" mz="1686.197995845770038" it="31099.400391" charge="29"/>
+				<element map="0" id="15057571094454825526" rt="226.647600000000011" mz="1528.188969366020046" it="23426.099609" charge="32"/>
+			</groupedElementList>
+			<UserParam type="string" name="label" value="T944"/>
```

Comment:
again, I dont think this is doing what https://github.com/OpenMS/OpenMS/issues/4073 intended: the meta data is still there!